data/raw/keybindings/messages.json

```
@@ -0,0 +1,16 @@
+[
+  {
+    "id": "TOGGLE_WIDE_DISPLAY",
+    "type": "keybinding",
+    "category": "MESSAGE_LOG",
+    "name": "Toggle wide display for messages",
+    "bindings": [ { "input_method": "keyboard", "key": "w" } ]
+  },
+  {
+    "id": "TOGGLE_FULL_HEIGHT_DISPLAY",
+    "type": "keybinding",
+    "category": "MESSAGE_LOG",
+    "name": "Toggle full height display for messages",
```

These are local to the message window, they don't need to be so verbose
```suggestion
    "name": "Toggle wide display",
    "bindings": [ { "input_method": "keyboard", "key": "w" } ]
  },
  {
    "id": "TOGGLE_FULL_HEIGHT_DISPLAY",
    "type": "keybinding",
    "category": "MESSAGE_LOG",
    "name": "Toggle full height display",
```